moviper/src/main/java/com/mateuszkoslacz/moviper/base/adapter/MoviperBaseRecyclerViewAdapter.java

```diff
@@ -0,0 +1,49 @@
+package com.mateuszkoslacz.moviper.base.adapter;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.ViewGroup;
+
+import com.hannesdorfmann.mosby.mvp.MvpPresenter;
+import com.hannesdorfmann.mosby.mvp.MvpView;
+import com.mateuszkoslacz.moviper.base.view.MoviperBaseViewHolder;
+
+/**
+ * Created by norbertbanaszek on 09.11.2016.
+ */
+
+public abstract class MoviperBaseRecyclerViewAdapter<V extends MvpView, P extends MvpPresenter<V>, VH extends MoviperBaseViewHolder<V, P>> extends RecyclerView.Adapter<VH> {
```

Comment:
Please stick to the project's line breaks convention and do not exceed 100 symbols line limit.